lib/osf-components/addon/components/resources-list/edit-resource/styles.scss

```diff
@@ -0,0 +1,3 @@
+.editResourceDialog {
```

Comment:
Same here for nomenclature for editResourceDialog -> EditResourceDialog or EditResource_dialog.